content/_code-samples/tx-serialization/js/main.js

```diff
@@ -0,0 +1,36 @@
+const { Client, encode } = require("xrpl");
+
+const payload = require("./json_input.json");
```

Comment:
It's worth bringing in the transaction formation within the snippet because ideally this code should run on it's own. Someone should be able to copy + paste it and start playing with it without any other setup.